public_html/layers.js

```diff
@@ -113,30 +113,27 @@ function createBaseLayers() {
                 }));
         }
 
-        if (ChartBundleLayers) {
-                var chartbundleTypes = {
-                        sec: "Sectional Charts",
-                        tac: "Terminal Area Charts",
-                        hel: "Helicopter Charts",
-                        enrl: "IFR Enroute Low Charts",
-                        enra: "IFR Area Charts",
-                        enrh: "IFR Enroute High Charts",
-                        secgrids: "Sect. w/ SAR grid"
-                };
+        if (FAALayers) {
+                const faa_tiles = [
+                        { name: 'VFR_Sectional', title: 'FAA VFR Sectional Chart' }, 
+                        { name: 'IFR_AreaLow', title: 'FAA IFR Area Low' },
+                        { name: 'IFR_High', title: 'FAA IFR High'},
+                ]
 
-                for (var type in chartbundleTypes) {
+                faa_tiles.forEach(tile => {
                         us.push(new ol.layer.Tile({
-                                source: new ol.source.TileWMS({
-                                        url: 'http://wms.chartbundle.com/wms',
-                                        params: {LAYERS: type},
-                                        projection: 'EPSG:3857',
-                                        attributions: 'Tiles courtesy of <a href="http://www.chartbundle.com/">ChartBundle</a>'
+                                source: new ol.source.XYZ({
```

Comment:
Should we use a TileArcGISRest layer here, not XYZ?

https://openlayers.org/en/latest/examples/arcgis-tiled.html

Reply:
I've tried the TileArcGISRest endpoint but for whatever reason, the server tells me the functionality is not supported.

GitHub won't let me upload an image for some reason.